tests/handlers/test_user_directory.py

```diff
@@ -797,43 +797,6 @@ async def block_all(user_profile: ProfileInfo) -> bool:
         s = self.get_success(self.handler.search_users(u1, "user2", 10))
         self.assertEqual(len(s["results"]), 0)
 
-    def test_legacy_spam_checker(self) -> None:
```

Comment:
I couldn't even figure out what this is attempting to test? It seems to be trying to test something from a really old way that modules were configured, but setting `spam_checker.spam_checker` is invalid?

Reply:
We can keep it, but we would need to adapt it and I couldn't figure out what a similar test is. I'll double check.